x/gamm/types/key_test.go

```diff
@@ -1,4 +1,4 @@
-package types
+package types_test
```

Comment:
Note to reviewer: This file is a drive by change. I realized it was a test file that wasn't getting run.